### PR TITLE
Nice repo

### PR DESCRIPTION
**Pull request by hulto**

### CODE REVIEW

**Comment by alexcote1** [comment]
wow, this needs to be pulled now. the prior version was just nothing compared to this fork. merge or i start using the superior fork

File: splunk-forwarders.md
Comment (MikeSE314) [inline splunk-forwarders.md:1]: This seems like a bad change.

File: splunk-forwarders.md [hunk 1]
```diff
@@ -1,93 +1,3 @@
-## Splunk Forwarder 9.0.1
-
-### Windows 10 , Windows 11, Windows Server 2012, 2012 R2, 2016, 2019, 2022 
-- [64-bit .msi](https://download.splunk.com/products/universalforwarder/releases/9.0.1/windows/splunkforwarder-9.0.1-82c987350fde-x64-release.msi) - [TinyURL](https://tinyurl.com/3scb529y)
+Nawww logs are bad.
+Not once. 😎
```
Comment (MikeSE314) [inline splunk-forwarders.md:2]: Unclear.

File: splunk-forwarders.md [hunk 2]
```diff
@@ -1,93 +1,3 @@
-## Splunk Forwarder 9.0.1
-
-### Windows 10 , Windows 11, Windows Server 2012, 2012 R2, 2016, 2019, 2022 
-- [64-bit .msi](https://download.splunk.com/products/universalforwarder/releases/9.0.1/windows/splunkforwarder-9.0.1-82c987350fde-x64-release.msi) - [TinyURL](https://tinyurl.com/3scb529y)
+Nawww logs are bad.
```
Comment (MikeSE314) [inline splunk-forwarders.md:1]: Citation needed.

**Review by MikeSE314 — CHANGES_REQUESTED** [review]
This change doesn't appear to be beneficial.

**Comment by kaisil** [comment]
Good ideas very good

**Comment by jgiboney** [comment]
Red team fun!

**Pull request closed without merging**